android/src/main/java/com/brentvatne/common/api/ControlsConfig.kt

```diff
@@ -9,6 +9,7 @@ class ControlsConfig {
     var hideDuration: Boolean = false
     var hideNavigationBarOnFullScreenMode: Boolean = true
     var hideNotificationBarOnFullScreenMode: Boolean = true
+    var liveLabel: String? = ""
```

Comment:
It should be better with this I think:
```suggestion
    var liveLabel: String? = null
```